### ISSUE TITLE
Custom Build Fails when no translation are listed

### ISSUE THREAD

**Issue by craigpotter** [FrDH/mmenu-js#982]
Hi, 

I'm using the latest version and when I try and run a custom build, it fails when no translations are supplied. 

```
[14:47:26] Starting 'jsTranslations'...
[14:47:26] 'jsTranslations' errored after 2.09 ms
[14:47:26] Error: Invalid glob argument:
    at Gulp.src (\custom-mmenu\node_modules\jquery.mmenu\node_modules\vinyl-fs\lib\src\index.js:20:11)
    at jsTranslations (\custom-mmenu\node_modules\jquery.mmenu\gulpfile.js:285:10)
    at bound (domain.js:396:14)
    at runBound (domain.js:409:12)
    at asyncRunner (\custom-mmenu\node_modules\jquery.mmenu\node_modules\async-done\index.js:55:18)
    at process._tickCallback (internal/process/next_tick.js:61:11)

```

Here is my build: 

```
{
	"name" 	: "jquery.mmenu.all",
	"umd"	: true,
	"files" : {
		"core" : [
			"oncanvas",
			"offcanvas",
			"scrollbugfix"
		],
		"extensions" : [
			"pagedim",
			"positioning"
		],
		"addons" : [
			"fixedelements",
			"iconbar",
			"navbars",
			"pagescroll"

		],
		"wrappers"     : [],
                "translations" : []
	}
}
```

If I add `"translations" : ["fr"]` it works fine.

**Comment by mrDinckleman** [comment]
Since 10 Jul 2015 empty array is not valid argument for gulp.src (https://github.com/micromatch/is-valid-glob/commit/e3ce0775bf3b05a2e6c67905c52ff20520ca9c8e)
You can try to use `gulp.src('.')` in https://github.com/FrDH/jQuery.mmenu/blob/master/gulpfile.js#L285

**Comment by FrDH** [comment]
Should be fixed in 7.3.3
(or you could simply include a language, it's only a couple of LOC extra)
